packages/chart-advisor/src/advisor/advice-pipeline/data-to-advices.ts

```diff
@@ -27,6 +29,7 @@ const scoreRules = (
   const showLog = options?.showLog;
   const purpose = options ? options.purpose : '';
   const preferences = options ? options.preferences : undefined;
+  const defaultWeight = DEFAULT_RULE_WEIGHTS;
```

Comment:
why not just use `DEFAULT_RULE_WEIGHTS` directly ?

Reply:
defaultWeight could be changed later... by user input